src/libcore/str/mod.rs

```diff
@@ -1150,7 +1150,7 @@ mod traits {
 
 /// Any string that can be represented as a slice
 #[unstable = "Instead of taking this bound generically, this trait will be \
-              replaced with one of slicing syntax, deref coercions, or \
+              replaced with one of slicing syntax (&foo[]), deref coercions (&*foo), or \
```

Comment:
This deref coercions is actually referring to [this RFC](https://github.com/rust-lang/rfcs/pull/241) rather than existing syntax. It may be worth calling it out a little more perhaps.